cache/options.go

```diff
@@ -25,3 +30,12 @@ func WithNegativeCaching(duration time.Duration) Option {
 		opts.Negative = duration
 	})
 }
+
+// WithShadowWrite configures the cache to use a separate Redis client for
+// shadow writes. This is useful for writing to a different Redis instance
+// than the one used for reading. This is useful for write-through caching
```

Comment:
```suggestion
// WithShadowWrite configures the cache to use a separate Redis client for
// shadow writes. This is useful for writing an additional copy of data to a
// different Redis instance than the primary instance used for caching.
```

I think these words are closer to what this does? I don't think this is "write-through caching", that's something else.

Reply:
good call, changed :)